README.md

```diff
@@ -89,6 +90,31 @@ iex> Litmus.validate(params, schema)
 {:error, "password must be alphanumeric"}
 ```
 
+### Litmus.Type.Number
+
+The `Number` module contains options that will validate Number data types. It converts "stringified" numerical values to numbers. It supports the following options:
+  * `:min` - Specifies the minimum value of the field. Allowed values are numbers.
+  * `:max` - Specifies the maximum value of the field. Allowed values are numbers.
+  * `:integer` - Specifies that the number must be an integer (no floating point). Allowed values are `true` and `false`. The default is `false`.
```

Comment:
Also, we'll allow the `required` param, correct?

Reply:
that's common to all modules. I have mentioned that in `Any` description. Should I explicitly mention in each module?